README.md

```diff
@@ -1,253 +1,239 @@
+A HarmonyX Mod aimed at improving the gameplay quality of Valheim. The mod includes several different main features that allow users to modify the stats of players, buildings and entities, and build and place objects with very high precision through a sophisticated system, while also allowing to modify already placed objects with equal precision. The goal is to provide V+ as a base modification for Valheim to increase quality of life, tweak the game's difficulty and in general, improve the player's experience. The mod also comes with a version and configuration control system for servers and users, enabling server owners to ensure that only players with the same configuration are able to join the server.
 
 # ValheimPlus Server Hosting
-Below you can find a list of hosting companies that are supporting ValehimPlus with good prices and easy installation.
+Below you can find a list of hosting companies that are supporting ValheimPlus with very competitive pricing and ease of use.
 
 [![GPortal](http://valheim.plus/gportal/banner.jpg)](http://gportal.valheim.plus/)
 [![ZapHosting](http://valheimplus.com/zap/692x127.jpg)](http://zap.valheim.plus/)
 [![GFXHosting](https://www.gtxgaming.co.uk/wp-content/uploads/2021/02/valheim_plus_banner-3.png)](http://gtxgaming.valheim.plus/)
 
 
-# Features
-All of these features can be adjusted by a configuration file. This also allows you to only enable very specific featues.
-
-
-## Player
-
-* Modification of stamina usage and regeneration
-* Modification of stamina usage of all tools and weaponry
-* Modification of carry weight
-* Modification of food duration
-* Modification of Unarmed Damage
-* Modification of auto pickup range
-* Option to force remove all screen shakes
-* Option to tweak/disable death penalty.
-* Option to disable food degradation.
-* Option to tweak Rested bonus duration per comfort level.
-* Modify the base amount of Unarmed damage multiplied by your unarmed skill level
-
-## Fermenter, Beehive, Kiln and Furnace
-* Modify Fermenter speed
-* Modify Fermenter output amount
-* Modify Beehive honey production speed
-* Modify Beehive maximum honey
-* Modify Furnace processing speed
-* Modify Furnace maximum coal/ore
-* Modify Kiln processing speed
-* Modify Kiln maximum wood
-* Show the amount of time left until the Fermenter is done.
-* Show the amount of time left until new honey is produced in a beehive.
-* Auto Deposit system for beehive honey to deposit it to the closest nearby chest.
-* Auto Deposit system for Furnace and Kiln to deposit items to the closest nearby chest.
-
-## Workbench and Ward
-* Modify Workbench radius
-* Modify Ward radius
-* Modify radius at which Workbench attachments can be placed.
-* Option to allow crafting stations to automatically repair all appropriate items in the player's inventory on interaction.
-
-## Fire sources
-* Disable all fire sources from running out of fuel once fuel is added.
```

Comment:
we should keep "once fuel is added" verbage in the new readme because this functionality is new for the fire feature in 0.9.5 and needs to be mentioned